### PR TITLE
Jr industrial auth

### PR DESCRIPTION
**Pull request by jb520**
Add authorization for comments and photos

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -63,6 +63,13 @@ def set_comment
       @comment = Comment.find(params[:id])
     end
 
+    def is_an_authorized_user
+      @photo = Photo.find(params.fetch(:comment).fetch(:photo_id))
+      if current_user != @photo.owner && @photo.owner.private? && !current_user.leaders.include?(@photo.owner)
```
Comment (MayaS1111) [inline app/controllers/comments_controller.rb:68]: Great job checking all cases

File: config/routes.rb
Comment (MayaS1111) [inline config/routes.rb:1]: Great use of only nad except

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job, Jennifer!

**Review by MayaS1111 — APPROVED** [review]
Good job!

**Pull request merged**